docs/encyclopedia/nexus-operations.mdx

```diff
@@ -0,0 +1,184 @@
+  - automatic retries
+  - execution semantics
+  - temporal sdk
+  - versioning
+  - nexus
+
+tags:
+  - Nexus
+  - Concepts
+
+---
+
+:::tip SUPPORT, STABILITY, and DEPENDENCY INFO
+
+Temporal Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for [Temporal Cloud](/cloud/nexus) and [self-hosted deployments](/production-deployment/self-hosted-guide/nexus).
+
+:::
+
+## Arbitrary-duration operations
+
+[Nexus Operations](/glossary#nexus-operation) are arbitrary-duration operations that may be synchronous or asynchronous, short-lived or long-lived, and are used to connect Temporal Applications within and across Namespaces, clusters, regions, and clouds.
+
+Unlike a traditional RPC, an asynchronous Nexus Operation has an operation identity that can be used to re-attach to a long-lived Nexus Operation, for example, one backed by a Temporal Workflow.
+Nexus Operations support a uniform interface to get the status of an operation or its result, receive a completion callback, or cancel the operation – all of which are fully integrated into the Temporal Platform.
+
+## SDK support
+
+The Temporal SDK provides an integrated Temporal experience to build, run, and use Nexus Operations.
+
+:::tip RESOURCES
+- [Go SDK - Nexus quick start and code sample](/develop/go/nexus)
+- [Java SDK - Nexus quick start and code sample](/develop/java/nexus)
+:::
+
+Caller Workflows use the Temporal SDK to execute Nexus Operations.
+
+Nexus Operation handlers are created with Temporal SDK builder functions such as:
+
+- New-Workflow-Run-Operation
+  - Start a Workflow as an asynchronous Operation.
+- New-Sync-Operation
+  - Invoke an underlying Query or Signal as a synchronous Operation.
+  - Invoke an Update as a synchronous Operation.
+  - Execute arbitrary code as a synchronous Operation.
+
+Nexus Operations are not intrinsically durable, but can be made durable by backing them with Temporal primitives like a Workflow, Update, or Signal.
+Operations may be implemented with Temporal primitives or execute arbitrary code, for example to make a database query or call a microservice.
+
+## Nexus Operation Lifecycle {#operation-lifecycle}
+
+When you execute a Nexus Operation from a caller Workflow using the Temporal SDK, a command is sent to Temporal to schedule the Nexus Operation which is atomically handed off to the Nexus Machinery.
+The [Nexus Machinery](/glossary#nexus-machinery) uses state-machine-based invocation and completion callbacks to ensure [at-least-once](#at-least-once-execution-semantics-and-idempotency) execution of a Nexus Operation and reliable delivery of the result.
+
+![Nexus Overview](/img/nexus-overview.png)
+
+The Nexus Machinery is responsible for making the Nexus RPC calls on behalf of the caller Workflow, with [automatic-retries](#automatic-retries).
+This means you don’t have to use Nexus RPC directly, only the Temporal SDK along with the Temporal Service.
+
+### System interactions
```

Comment:
Arguably most of this section is redundant given that you're explaining the sync and async lifecycle below.

Reply:
it's more of an intro/overview and have added different diagrams for each section, which hopefully make it easier to understand for readers not familiar with it.